app/request/views.py

```diff
@@ -114,7 +114,7 @@ def new():
             request_id = create_request(form.request_title.data,
                                         form.request_description.data,
                                         category=None,
-                                        agency_ein=current_user.agency_ein,
+                                        agency_ein=form.request_agency.data,
```

Comment:
This does not account for if the current agency user is not part of multiple agencies. If they submit a request, `form.request_agnecy.data` evaluates to None.